### PR TITLE
{data}[GCCcore/11.2.0] dill v0.3.4 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3605.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/5da723a7958efe73c48f49613482851a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3144.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/6da0dd7e5d3bd26c5c1cdf83f17370cd for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/d/dill/dill-0.3.4-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
```
Comment (lexming) [inline easybuild/easyconfigs/d/dill/dill-0.3.4-GCCcore-11.2.0.eb:1]: Previous version `dill-0.3.4-GCCcore-10.3.0.eb` has an attribution in the header. Please keep it in this update.

**Review by lexming — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/d/dill/dill-0.3.4-GCCcore-11.2.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/d/dill/dill-0.3.4-GCCcore-11.2.0.eb:2]: @lexming fixed in e486f98357

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**